synthesizer/process/src/stack/helpers/initialize.rs

```diff
@@ -23,12 +23,15 @@ impl<N: Network> Stack<N> {
         let mut stack = Self {
             program: program.clone(),
             stacks: Arc::downgrade(&process.stacks),
+            constructor_types: Default::default(),
             register_types: Default::default(),
             finalize_types: Default::default(),
             universal_srs: process.universal_srs().clone(),
             proving_keys: Default::default(),
             verifying_keys: Default::default(),
             program_address: program.id().to_address()?,
+            program_checksum: program.checksum()?,
+            program_edition: U16::new(N::EDITION),
```

Comment:
Will we need migration logic for disallowing deployments with old editions once `Network::EDITION` is updated?

Reply:
With updatability, deployments will no longer use `N::EDITION`. Instead, they will start from 0 and increment.
Because `N::EDITION` is 0 for all current networks, I don't believe we will need migration logic.